asn1-core/src/types/octet_string.rs

```diff
@@ -2,15 +2,27 @@ use std::ops::{Deref, DerefMut};
 
 use serde::{Deserialize, Serialize};
 
+/// A representation of the `OCTET STRING` ASN.1 data type.  Use an
+/// `OCTET STRING` type to model binary data whose format and length are
+/// unspecified, or specified elsewhere, and whose length in bits is a multiple
+/// of eight.
+/// # Example
+/// ```asn1
+/// G4FacsimileImage ::= OCTET STRING
+/// -- a sequence of octets conforming to Rec. ITU-T T.5 and CCITT Rec. T.6
+/// image G4FacsimileImage ::= '3FE2EBAD471005'H
+/// ```
 #[derive(Debug, Default, Clone, Deserialize, Serialize, PartialEq)]
 #[serde(rename = "ASN.1#OctetString")]
 pub struct OctetString(Vec<u8>);
 
 impl OctetString {
+    /// Instantiate an empty instance of `OctetString`.
```

Comment:
This could be simplified as:

```suggestion
    /// Creates an empty `OctetString`.
```